package.json

```diff
@@ -53,7 +53,7 @@
     "react-native-music-control": "^0.11.0",
     "react-native-permissions": "^3.0.0",
     "react-native-push-notification": "^3.1.9",
-    "react-native-static-server": "^0.5.0",
+    "react-native-static-server": "file:react-native-static-server.tgz",
```

Comment:
you'll want to commit the tgz file for this to work, but i think i do have a local repro & would be curious to test w/ this package update

Reply:
the file is committed indeed :)